src/main/java/org/folio/rmapi/model/Error.java

```diff
@@ -0,0 +1,26 @@
+package org.folio.rmapi.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Error {
+  @JsonProperty("code")
+  private String code;
+  @JsonProperty("message")
+  private String message;
+  @JsonProperty("subCode")
+  private String subCode;
+
+  public String getCode() {
```

Comment:
I don't think we are interested in "code" and "subCode" returned by RM API - just the error message in some cases

Reply:
RM API returns code and subcode, here's example
```
{
  "errors": [
    {
      "code": "1006",
      "message": "Parameter 'count' is outside the range 1-100.",
      "subCode": "0"
    }
  ]
}
```
it is taken from here 
https://developer.ebsco.com/interactive/holdingsiq#/

Reply:
I think its ok if the codes appear in these objects since they are used to serialize the results from rm api response. When the error is built from rm api response (in ErrorUtil.java) we are only using the message

Reply:
sounds good 👍